Dfe.Academies.External.Web/Pages/ApplicationOverview.cshtml.cs

```diff
@@ -116,7 +123,18 @@ private void PopulateUiModel(ConversionApplication? conversionApplication)
 		{
 			
 			// grab current user email
-			string email = User.FindFirst(ClaimTypes.Email)?.Value ?? "";
+			string email = User.FindFirst(ClaimTypes.Email)?.Value ?? "";		
+			var firstContributorEmail = conversionApplication.Contributors.FirstOrDefault().EmailAddress;
+			
+		    if(email == firstContributorEmail)
+			{
+				HideDeleteButton = false;
+			}
+
+			else
```

Comment:
Don't need the else - I find avoiding else's usually leads to more performant code - this one is a small one but I'd do
`HideDeleteButton = true

if(email is firstContributorEMail) HideDeleteButton = false`